SeaPkg/Tools/GenSeaArtifacts/gen_aux/src/report.rs

```diff
@@ -82,6 +88,58 @@ impl CoverageReport {
     }
 }
 
+/// A wrapper around a list of sections that allows for inserting new sections into the list.
+struct SectionList {
+    size_of_image: u32,
+    sections: Vec<Section>,
+}
+
+impl SectionList {
+    /// Create an empty section list with the given image size.
+    pub fn new(size: u32) -> Self {
+        SectionList {
+            size_of_image: size,
+            sections: Vec::new(),
+        }
+    }
+
+    /// Consumes the list and returns the inner vector of sections.
+    pub fn into_inner(self) -> Vec<Section> {
+        self.sections
+    }
+
+    /// Adds a new section to the list with the given name, start address, and end address.
+    fn add_section(&mut self, name: String, start: u32, end: u32) {

```

Comment:
The image size as a 4-byte value makes sense because `SizeOfImage` is 4-bytes in both PE32/PE32+. But the addresses could be 64-bit, right?

Reply:
The `VirtualAddress`, field in a Section Header of a PE image is 4-bytes. While there is a possibility that virtual address + size can overflow, that would theoritically break the next sections `VirtualAddress` field, so I think its OK.

Reply:
All the data I'm getting from the pdb parser is u32. I'm not doing any type of `usize` to `u32` conversions either

Reply:
If this does not include the base (RVA), I agree.